### PR TITLE
[tests-only][full-ci] Add tests to list same name resources shared by user from two project-spaces with him/her

### PR DESCRIPTION
**Pull request by nabim777**
## Description

In this PR following scenarios are added related to sharingNG:
```
- user lists the file with same name shared by user from two project-spaces with him/her
- user lists the folder with same name shared by user from two project-spaces with him/her
```

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/8318

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [X] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature [hunk 1]
```diff
@@ -2794,3 +2794,522 @@ Feature: listing sharedWithMe when auto-sync is disabled
...
+                          "type": "object",
+                          "required": ["user"],
+                          "properties": {
+                            "user": {
+                              "type": "object",
+                              "required": ["displayName","id"],
+                              "properties": {
+                                "displayName": {
+                                  "type": "string",
+                                  "enum": ["Carol King"]
```
Comment (PrajwolAmatya) [inline tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature:3118]: Carol ?

**Comment by saw-jan** [comment]
Please, make changes to json schema
```
'items' should be an object when expecting 1 element
Failed asserting that false is true.
```

File: tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature [hunk 2]
```diff
@@ -2814,3 +2814,626 @@ Feature: listing sharedWithMe when auto-sync is disabled
...
+                  "required": [
+                    "@UI.Hidden",
+                    "@client.synchronize",
+                    "eTag",
+                    "file",
+                    "id",
+                    "lastModifiedDateTime",
+                    "name",
+                    "parentReference",
+                    "remoteItem"
```
[the remark this answers is not in the document]
Comment (nabim777) [reply tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature]: okay

File: tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature [hunk 3]
```diff
@@ -2814,3 +2814,626 @@ Feature: listing sharedWithMe when auto-sync is disabled
...
+                      "properties": {
+                        "driveId": {
+                          "type": "string",
+                          "pattern": "^%space_id_pattern%$"
+                        },
+                        "driveType": {
+                          "type": "string",
+                          "enum": [
+                            "virtual"
+                          ]
```
[the remark this answers is not in the document]
Comment (nabim777) [reply tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature]: okay

File: tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature [hunk 4]
```diff
@@ -2814,3 +2814,622 @@ Feature: listing sharedWithMe when auto-sync is disabled
         }
       }
       """
+
+
+  Scenario: user lists the files with same name shared by user from two project-spaces with him/her
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature:2819]:
```suggestion
  Scenario: sharee lists the files with same name shared from different project-spaces
```

File: tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature [hunk 5]
```diff
@@ -2814,3 +2814,622 @@ Feature: listing sharedWithMe when auto-sync is disabled
...
+                                      "required": ["displayName", "id"],
+                                      "properties": {
+                                        "displayName": {
+                                          "const": "Brian Murphy"
+                                        }
+                                      }
+                                    }
+                                  }
+                              },
+                              "invitation": {
```
Comment (PrajwolAmatya) [inline tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature:3291]: The indentation looks off. Can you check?

**Review by saw-jan — APPROVED** [review]
LGTM

**Review by PrajwolAmatya — APPROVED** [review]

**Pull request merged**